### DISCUSSION TITLE
Github Token & Pycharm

### DISCUSSION THREAD

**Discussion by Moviick** [community/community#39596] [New to GitHub] [answered]
New to github…
When try to get Github and Pycharm talk to each other, but Pycharm asks for the server, the password and a “token”. What token are they referring to? I tried the FAQ, but it’s not clear to me.
Thanks

**Comment by BREN0sx** [comment, marked as answer]
Maybe you have to create a personal token in github. Follow [this link](https://github.com/settings/tokens) then click to “Generate new token”, give a name to the token and you have your code. Now go to Pycharm settings, find github and add your user credentials.
Or you can do only with settings->github->add your user credentials. Then if you go to https://github.com/settings/tokens you can find a new token with name “intelliJ plugin” or something like that. 

This can solve your problem… I hope 😃

**Reply by Moviick** [reply]
Thanks! 💪🙏

**Reply by queenofcorgis** [reply]
@BREN0sx thanks for the links and instructions - you rock 🚀 !

**Comment by Mikem2211** [comment]
Thanks
